kilted/distribution.yaml

```diff
@@ -5960,13 +5960,13 @@ repositories:
     source:
       type: git
       url: https://github.com/open-rmf/rmf_building_map_msgs.git
-      version: main
+      version: kilted
     status: developed
   rmf_cmake_uncrustify:
     doc:
       type: git
       url: https://github.com/open-rmf/rmf_cmake_uncrustify.git
-      version: rolling
+      version: kilted
```

Comment:
CI is complaining that this repo doesn't have a `kilted` branch

Reply:
It looks like this repo hasn't seen any changes in 4 years. We can probably just keep it on `rolling`.

Reply:
We should probably just remove that repo from Rolling since it was a temporary package and changes from it landed in `ament_uncrustify`.